data/polkit/io.snapcraft.snapd.policy

```diff
@@ -26,4 +26,15 @@
       <allow_active>auth_admin_keep</allow_active>
     </defaults>
   </action>
+
+  <action id="io.snapcraft.snapd.manage-interfaces">
+    <description>Connect, disconnect interfaces </description>
```

Comment:
Don't know if trailing whitespace matters, but the description is: `"Connect, disconnect interfaces "`